### PR TITLE
PR Audit: Add default colors to created labels

### PR DESCRIPTION
**Pull request by gnarf**

### CODE REVIEW

File: lib/pr.js [hunk 1]
```diff
@@ -79,14 +79,14 @@ Audit.prototype.ensureClaLabels = function() {
...
+		return repo.createLabel({ name: name, color: color });
 	}
 
 	return this.repo.getLabels()
 		.then(function( labels ) {
 			return Promise.all([
-				ensureLabel( labels, "CLA: Valid" ),
-				ensureLabel( labels, "CLA: Error" )
+				ensureLabel( labels, "CLA: Valid", "#007700" ),
+				ensureLabel( labels, "CLA: Error", "#770000" )
```
Comment (scottgonzalez) [inline lib/pr.js]: Just the hex value, no hash in front.

Comment (gnarf) [inline lib/pr.js]: fixed

**Comment by gnarf** [comment]
hrm, no commits audited?

http://contribute.jquery.org/CLA/status/?repo=jquery-license&sha=312d6add046d2e3adde14969aea5ca14116d0386

**Comment by gnarf** [comment]
Hrm -- we might want to make the bot only add/remove labels when they aren't already on the issue?  It's pretty spammy if it adds a note about it for every rebase commit :)

**Comment by scottgonzalez** [comment]
Filed #11 for less spammy labeling.

**Pull request closed without merging**